diff --git a/dist/index.mjs b/dist/index.mjs
index 6fec988ac4ea9d84011934f4ac235485a5732353..45de427d865381553242865d26b8611045c9f17d 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -51,13 +51,12 @@ function ViteSSG(App, routerOptions, fn, options) {
     app.use(router);
     let entryRoutePath;
     let isFirstRoute = true;
-    router.beforeEach((to, from, next) => {
+    router.beforeEach((to, from) => {
       if (isFirstRoute || entryRoutePath && entryRoutePath === to.path) {
         isFirstRoute = false;
         entryRoutePath = to.path;
         to.meta.state = context.initialState;
       }
-      next();
     });
     if (import.meta.env.SSR) {
       const route = context.routePath ?? "/";
